<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowWidget</class>
 <widget class="QMainWindow" name="MainWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_openAction"/>
    <addaction name="m_writeAction"/>
    <addaction name="separator"/>
    <addaction name="m_loadEvents"/>
    <addaction name="m_saveEvents"/>
    <addaction name="separator"/>
    <addaction name="m_quitAction"/>
   </widget>
   <widget class="QMenu" name="menuAdjust">
    <property name="title">
     <string>Adjust</string>
    </property>
    <addaction name="m_filterAction"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="m_eventAction"/>
    <addaction name="m_informationAction"/>
    <addaction name="m_channelSelectionManagerAction"/>
    <addaction name="m_averageWindowAction"/>
    <addaction name="m_scalingAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m_aboutAction"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuAdjust"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_openAction">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_writeAction">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_loadEvents">
   <property name="text">
    <string>Load Events (fif)...</string>
   </property>
  </action>
  <action name="m_saveEvents">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Events (fif)...</string>
   </property>
  </action>
  <action name="m_quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="m_filterAction">
   <property name="text">
    <string>Filter...</string>
   </property>
  </action>
  <action name="m_dataAction">
   <property name="text">
    <string>Show data plot...</string>
   </property>
  </action>
  <action name="m_eventAction">
   <property name="text">
    <string>Show event list...</string>
   </property>
  </action>
  <action name="m_informationAction">
   <property name="text">
    <string>Show information...</string>
   </property>
  </action>
  <action name="m_aboutAction">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="m_channelSelectionManagerAction">
   <property name="text">
    <string>Show channel selection manager...</string>
   </property>
  </action>
  <action name="m_averageWindowAction">
   <property name="text">
    <string>Show averages...</string>
   </property>
  </action>
  <action name="m_scalingAction">
   <property name="text">
    <string>Show scaling...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>openFile()</slot>
  <slot>writeFile()</slot>
  <slot>loadEvents()</slot>
  <slot>saveEvents()</slot>
  <slot>quit()</slot>
  <slot>showFilterWindow()</slot>
  <slot>showEventWindow()</slot>
  <slot>showDataWindow()</slot>
  <slot>showLogWindow()</slot>
  <slot>about()</slot>
 </slots>
</ui>
